smarty/templates/install.tpl

```diff
@@ -151,6 +165,41 @@
 					<input id="dbname" value="{if $dbname}{$dbname}{else}LORIS{/if}" type="text" name="dbname">
 				</div>
 			</div>
+            <hr/>
+			<div>
+				<div class="col-md-2">
+					<label for="use_existing_database">Use Existing Database:</label>
+				</div>
+				<div class="col-md-9">
+					<input type="checkbox" id="use_existing_database" name="use_existing_database" value="on" {($use_existing_database == 'on') ? 'checked' : ''}/>
+                    Check this if the database already exists but does not have Loris installed yet
+                    <!--The following is a hack to get the UI to align correctly *shrug*-->
```

Comment:
This should be taken out. (The hack if possible, and the "*shrug*" if fixing the hack is more effort than it's worth.)

(Same goes for the other two instances below.)